hw/top_earlgrey/BUILD

```diff
@@ -120,7 +123,10 @@ fpga_cw305(
     name = "fpga_cw305",
     design = "earlgrey",
     exec_env = "fpga_cw305",
-    lib = "//sw/device/lib/arch:fpga_cw305",
+    libs = [
+        "//sw/device/lib/arch:fpga_cw305",
+        "//hw/top_earlgrey/sw/dts:cw310_devicetree",
```

Comment:
These of course need to change for this board. I'm just showing the flow for now.